modules/test/playwright/tests/frontend-data-set-views-web/sorting.spec.ts

```diff
@@ -0,0 +1,169 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {featureFlagsTest} from '../../fixtures/featureFlagsTest';
+import {loginTest} from '../../fixtures/loginTest';
+import getRandomString from '../../utils/getRandomString';
+import {dataSetManagerApiHelpersTest} from './fixtures/dataSetManagerApiHelpersTest';
+import {dataSetsPageTest} from './fixtures/dataSetsPageTest';
+import {sortingPageTest} from './fixtures/sortingPageTest';
+import {viewsPageTest} from './fixtures/viewsPageTest';
+
+export const test = mergeTests(
+	dataSetManagerApiHelpersTest,
+	dataSetsPageTest,
+	featureFlagsTest({
+		'LPD-10735': false,
+		'LPS-164563': true,
+		'LPS-178052': true,
+		'LPS-186871': true,
+	}),
+	sortingPageTest,
+	loginTest(),
+	viewsPageTest
+);
+
+let sortingDataSetERC: string;
+let sortingDataSetLabel: string;
+let sortingDataSetViewERC: string;
+let sortingDataSetViewLabel: string;
+
+test.beforeEach(async ({dataSetManagerApiHelpers}) => {
+	sortingDataSetERC = getRandomString();
+	sortingDataSetLabel = getRandomString();
+	sortingDataSetViewERC = getRandomString();
+	sortingDataSetViewLabel = getRandomString();
+
+	await dataSetManagerApiHelpers.createDataSet({
+		erc: sortingDataSetERC,
+		label: sortingDataSetLabel,
+	});
+	await dataSetManagerApiHelpers.createDataSetView({
+		erc: sortingDataSetViewERC,
+		label: sortingDataSetViewLabel,
+		r_fdsEntryFDSViewRelationship_c_fdsEntryERC: sortingDataSetERC,
+	});
+});
+
+test.afterEach(async ({dataSetManagerApiHelpers}) => {
+	await dataSetManagerApiHelpers.deleteDataSet({erc: sortingDataSetERC});
+});
+
+test.describe('Data Set Sorting', () => {
+	test.slow();
```

Comment:
I don't think this is the right solution. Looking at [docs](https://playwright.dev/docs/api/class-test#test-slow), this triples timeouts for this tests, but I don't think timeout is the problem. Default timeout in our tests is [15 seconds](https://github.com/liferay/liferay-portal/blob/master/modules/test/playwright/playwright.config.ts#L42), that should be more then enough.

The problem is that when `await this.addSortingButton.click();` tries to resolve, there is no button in UI to click on. Locally, for you, the page loaded, but for me in https://github.com/liferay-frontend/liferay-portal/pull/4085#pullrequestreview-1994000112, the page still only contained a spinner. One would expect that Playwright will hold off with click for 15 seconds, but that's not how it works. Maybe it's because we're chaining locator to `click`, so `await` only applies to `click`? :thinking:  A solution we previously did is to explicitly wait until page loads, like [here](https://github.com/liferay/liferay-portal/blob/master/modules/test/playwright/tests/frontend-data-set-views-web/visualizationModes.spec.ts#L66-L70), checking if something on the page post-spinner is visible, or in viewport. We could await for the button itself, separating `await this.addSortingButton.click();` into two steps.